README.md

```diff
@@ -11,6 +11,13 @@ These are regexes that could lead to [catastrophic backtracking](https://www.reg
 2. Run the `configure` script to install dependencies and build the detectors.
 3. Use the scripts in `bin`. See their [README](https://github.com/davisjam/vuln-regex-detector/blob/master/bin/README.md) for details.
 
+Alternately, a `Dockerfile` is provided to allow the project to run on non-Ubuntu systems. The image can be built and used as follows:
```

Comment:
Can you move the Docker info to the "Supported OSes" section and rewrite that text to say "Ubuntu is supported directly, for anything else use a container"?

Reply:
Moved and updated.